trymito.io/components/InstallInstructions/InstallInstructions.tsx

```diff
@@ -15,16 +16,17 @@ const InstallInstructions = (props: {}): JSX.Element => {
                 in under a minute
             </h2>
             <div className={installInstructions.install_instructions_container}>
-                <CodeBlock prefix='$ ' paddingRight='7rem'>
+                <CodeBlock prefix='$ ' paddingRight='7rem' className={PLAUSIBLE_COPIED_PIP_INSTALL_COMMAND}>
                     pip install mitosheet
                 </CodeBlock>
-                <CodeBlock prefix='$ ' paddingRight='7rem'>
+                <CodeBlock prefix='$ ' paddingRight='7rem' className={PLAUSIBLE_COPIED_MITOSHEET_HELLO_COMMAND}>
                     python -m mitosheet hello
                 </CodeBlock>
                 <div className={ctaButtons.cta_subbutton}>
                     <Link href={MITO_INSTALLATION_DOCS_LINK}>
```

Comment:
1. This should probably take you to the `create mitosheet` page instead of the install page?
2. Should we say something in addition like "If your install failed, try using these common installation issues fixes"
3. It would be good to link our Discord/Slack here so users can get support if they need since we don't have the the ability to show it if the pip install failed

Reply:
1. Yes
2. No for now. 
3. Added